http-gateway/service/updateDeviceMetadata_test.go

```diff
@@ -63,9 +63,10 @@ func (f *contentChangedFilter) Handle(ctx context.Context, iter eventbus.Iter) (
 			if err != nil {
 				return err
 			}
-			if ev.GetShadowSynchronization() == commands.ShadowSynchronization_UNSET {
-				continue
-			}
+			// TODO ????
```

Comment:
remove comment or finish code if something is missing

Reply:
removed